package.json

```
@@ -14,6 +14,7 @@
     "lint": "npm run lint:eslint && npm run lint:lit",
```

would add an extra check for style lints so that testing environment will simulate github checks 

```suggestion
    "lint": "npm run lint:eslint && npm run lint:lit && npm run lint:style",
```